src/components/common/PageAbout.vue

```diff
@@ -43,7 +43,12 @@
       <p>
         To send transactions with Lunie, you'll have to sign them with your
         Ledger Nano&nbsp;S. If you don't have a Ledger Nano S, you can
-        <a href="" target="_blank" rel="noopener norefferer">buy one here</a>.
+        <a
+          href="https://shop.ledger.com/products/ledger-nano-s"
```

Comment:
```suggestion
          href="https://shop.ledger.com/?r=3dd204ef7508"
```

Reply:
Should we just refer to the generic 'Ledger', or 'Ledger Nano`?

Reply:
i think we can say "Ledger Nano" for now on - since we support S and X - what do you think?

Reply:
Sure 👍